lux/extensions/ui/css/navbar.py

```diff
@@ -15,67 +15,100 @@ def add_css(all):
     # STYLING
     navbar = vars.navbar
     #
+    navbar.transition.duration_in = '0.6s'
+    navbar.transition.duration_out = '0.2s'
+    navbar.transition.duration = '0.8s'
+    navbar.collapse.background = '#eaeaea'
```

Comment:
why do we need to specify the color?
What about if we switch to a different bootstrap theme?